Fixes page unlock button styling.

BertrandBordage opened the pull request:
This pull request fixes the clickable area of the page unlock button. Before this pull request:
- we could click only on the text of the button, not on all the visible area
- the text got an ugly background on hover

## Before
![selection_007](https://user-images.githubusercontent.com/1119169/34022093-8556d424-e13d-11e7-8608-f0f7bf36d821.png)
![selection_008](https://user-images.githubusercontent.com/1119169/34022094-857a9a44-e13d-11e7-8c42-8fd79e177b3e.png)
## After
![selection_009](https://user-images.githubusercontent.com/1119169/34022095-85a2983c-e13d-11e7-8157-30351b99c86a.png)
![selection_010](https://user-images.githubusercontent.com/1119169/34022096-85c12252-e13d-11e7-90b9-432103c594b2.png)

jjanssen commented:
Hi @BertrandBordage, I noticed the rendering of the button is a bit off in Firefox ESR and the latest version. Would you mind taking a look at that? Otherwise this change is good to go.

![image](https://user-images.githubusercontent.com/287422/34032280-9cb794c2-e175-11e7-9e2f-2c7a22fd9a25.png)

BertrandBordage commented:
Thanks for testing @jjanssen!
I fixed the Firefox rendering. I also fixed the button when the user has no locking permission, I broke it in the initial commit, it was not correctly disabled.
